src/displayapp/screens/settings/SettingWatchFace.cpp

```diff
@@ -5,7 +5,9 @@
 #include "displayapp/screens/Screen.h"
 #include "displayapp/screens/Styles.h"
 #include "displayapp/screens/Symbols.h"
+#include "displayapp/WatchFaces.h"
```

Comment:
two times the same include, remove one of them please

Reply:
Thank you!